auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/HealthCheckController.java

```diff
@@ -0,0 +1,15 @@
+package com.atlantbh.auctionappbackend;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
```

Comment:
Add some route for health check API (e.g. `/health`). Also, to make it easier to manage the versions of APIs, this and all other APIs could be prefixed by `/api/v1/`.

Reply:
Sure.